packages/base-controller/src/index.ts

```diff
@@ -1,17 +1,17 @@
-export type { BaseConfig, BaseState, Listener } from './BaseController';
-export { BaseController } from './BaseController';
+export type { BaseConfig, BaseState, Listener } from './BaseControllerV1';
+export { BaseController as BaseControllerV1 } from './BaseControllerV1';
```

Comment:
What do you think about renaming `BaseController` to `BaseControllerV1` in `BaseControllerV1.ts`? That way it matches the file name and we don't have to alias it here.

Reply:
We can definitely do that. Applied here: https://github.com/MetaMask/core/pull/2078/commits/42a87e781b2f5a1451a17529c2e2462d14434a99